src/bin/pgcopydb/catalog.c

```diff
@@ -243,6 +252,15 @@ static char *filterDBcreateDDLs[] = {
 	"create unique index s_t_qname on s_table(qname)",
 	"create unique index s_t_rlname on s_table(restore_list_name)",
 
+	"create table s_matview("
+	"  oid integer primary key, "
+	"  qname text, nspname text, relname text, restore_list_name text, "
+	"  exclude_data boolean"
+	")",
+
+	"create unique index s_mv_rlname on s_matview(restore_list_name)",
+	"create unique index s_mv_qname on s_matview(nspname, relname)",
+
```

Comment:
I am not seeing where this is filled, do we still use it in the PR?

Reply:
It would be populated when table complement filter operation runs (i.e. exclude-table or include-only-table)., i.e. the same function which populates this table on source.db also populates for filter.db during complement operation.